File: backend/internconnectbackendspring/src/main/java/com/internconnect/internconnectbackendspring/model/Student.java
package com.internconnect.internconnectbackendspring.model;

import jakarta.persistence.*;

import java.io.Serializable;
@Entity
@Table(name = "student")
public class Student implements Serializable {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    @Column(name = "email", nullable = false, unique = true, length = 255)
    private String email;
    @Column(name = "pass", nullable = false, length = 255)
    private String pass;
    @Column(name = "firstName", nullable = false, length = 255)
    private String firstName;
    @Column(name = "lastName", nullable = false, length = 255)
    private String lastName;
    @Column(name = "school", nullable = false, length = 255)
    private String school;
    @Column(name = "yearOfStudy", nullable = false, length = 255)
    private String yearOfStudy;
}

File: frontend/internconnectfrontendclient/composeApp/src/commonMain/kotlin/com/lazar/internconnectfrontendclient/data/model/Student.kt
package com.lazar.internconnectfrontendclient.data.model

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity
data class Student(
    @PrimaryKey(autoGenerate = true)
    val id: Long = 0,
    @ColumnInfo(name = "email")
    val email: String,
    @ColumnInfo(name = "pass")
    val pass: String,
    @ColumnInfo(name = "firstName")
    val firstName: String,
    @ColumnInfo(name = "lastName")
    val lastName: String,
    @ColumnInfo(name = "school")
    val school: String,
    @ColumnInfo(name = "yearOfStudy")
    val yearOfStudy: Long
)